### PR TITLE
feat(http): Add Sec-WebSocket-* headers

### PR DESCRIPTION
**Pull request by bsmth**
Adding data in for `Sec-WebSocket-*` headers. I am copying over info from `api.WebSocket.protocol_rfc_6455` -> https://github.com/mdn/browser-compat-data/blob/7f9f083afb80c0777617d96e06283c15f43f39b3/api/WebSocket.json#L673

#### Test results and supporting details

I began testing using BrowserStack, but support goes so far back, it's probably easier to mirror the rfc_6455 compliance entry instead.

__Other changes:__

- typo fixes in parallel

#### Related issues and pull requests

- [x] https://github.com/mdn/content/pull/36590

### CODE REVIEW

**Comment by caugner** [comment]
@bsmth We discussed the failing check in yesterday's BCD call, and the recommendation is to file an issue in [w3c/browser-specs](https://github.com/w3c/browser-specs) to add this RFC.

**Comment by bsmth** [comment]
Changes to browser-specs incoming in next release:

- [x] https://github.com/w3c/browser-specs/pull/1575

File: css/properties/font-variant-position.json
Comment (caugner) [inline css/properties/font-variant-position.json:1]: Nit: Extract these unrelated fixes to a separate PR.

File: api/SVGGraphicsElement.json [hunk 1]
```diff
@@ -117,7 +117,7 @@
             "firefox": {
               "version_added": "1.5",
               "notes": [
-                "Before Firefox 128, the `getBBox()` method retured an empty `DOMRect` when there is no fill ([bug 1019326](https://bugzil.la/1019326)).",
+                "Before Firefox 128, the `getBBox()` method returned an empty `DOMRect` when there is no fill ([bug 1019326](https://bugzil.la/1019326)).",
```
Comment (caugner) [inline api/SVGGraphicsElement.json:120]: Nit: Extract these unrelated fixes to a separate PR.

**Review by caugner — APPROVED** [review]
LGTM, and copying from RFC 6455 compliance is plausible, but I would prefer to extract the unrelated fixes to a separate PR (it'll be a no-brainer to approve).

File: api/SVGGraphicsElement.json (hunk 1 above)
Comment (bsmth) [reply api/SVGGraphicsElement.json:120]: No problem 👍🏻

File: css/properties/font-variant-position.json
Comment (bsmth) [reply css/properties/font-variant-position.json:1]: done in cb724cd33192e02a83735d34bfcaa9b6bb4367fe

File: api/SVGGraphicsElement.json (hunk 1 above)
Comment (bsmth) [reply api/SVGGraphicsElement.json]: done in cb724cd33192e02a83735d34bfcaa9b6bb4367fe

File: css/properties/font-variant-position.json
Comment (bsmth) [reply css/properties/font-variant-position.json:1]: opened in follow-up 

- [ ] https://github.com/mdn/browser-compat-data/pull/25180

**Review by caugner — APPROVED** [review]
